### DISCUSSION TITLE
Verification and reversion for UI.Select

### DISCUSSION THREAD

**Discussion by vickorian** [zauberzeug/nicegui#4031] [Q&A] [answered]
### Question

I have a UI.Select element that controls the current client target. The page has all the information regarding that target, but when that selection changes the page refreshes to generate the new values and re-run some on-load functions that can't easily be done without it. The question i have though, is that is there a way to stop UI.Select from making the change without a verification box?

I could write a function to do the dialog box, ask if they want to proceed and if they do refresh the page, and if they don't reset the value to the original value. I just wanted to see if there was anything built into UI.Select that allowed for this before i hacked something together.

**Comment by rodja** [comment, marked as answer]
`ui.select` does not have a build in conformation mechanism. As you suggested this kind of behaviour can (and should) be implemented in your code.

**Reply by vickorian** [reply]
@rodja thanks
